src/java/com/palantir/cassandra/utils/RangeTombstoneCountingIterator.java

```diff
@@ -23,6 +23,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.codahale.metrics.Snapshot;
 import org.apache.cassandra.db.ColumnFamily;
```

Comment:
nit: unused import

Reply:
Will update.

Is there a trick to getting this repo to play nicely with IntelliJ to autofix imports? I assume we don't want to apply the Palantir-java-format as Cassandra has a very different source style and backports would be much more difficult.

Reply:
updated